keystone.js

```diff
@@ -0,0 +1,65 @@
+// Simulate config options from your production environment by
+// customising the .env file in your project's root folder.
+require('dotenv').config();
+
+// Require keystone
+var keystone = require('keystone');
+var handlebars = require('express-handlebars');
+
+// Initialise Keystone with your project's configuration.
+// See http://keystonejs.com/guide/config for available options
+// and documentation.
+
+keystone.init({
+	'name': 'Midwest PHP',
+	'brand': 'Midwest PHP',
+
+	// 'sass': 'public',
+	'static': 'public',
+	'favicon': 'public/favicon.ico',
+	'views': 'templates/views',
+	'view engine': '.hbs',
+
+	'custom engine': handlebars.create({
+		layoutsDir: 'templates/views/layouts',
+		partialsDir: 'templates/views/partials',
+		defaultLayout: 'default',
+		helpers: new require('./templates/views/helpers')(),
+		extname: '.hbs',
+	}).engine,
+
+	'auto update': true,
+	'session': true,
+	'auth': true,
+	'user model': 'User',
+});
+
+// Load your project's Models
+keystone.import('models');
+
+keystone.mongoose.set('debug', true);
```

Comment:
Set this based on the environmental .env configuration.

Reply:
Removed the code as it was only being used when I was testing locally previously.